cmd/gokeyless-cli-tester/gokeyless-cli-tester.go

```diff
@@ -50,6 +60,55 @@ func main() {
 	if err != nil {
 		log.Fatal("failed to load cert directory:", err)
 	}
+	if delegate {
+		r, err := c.LookupServer(keyserver)
+		if err != nil {
+			log.Fatal("Could not look up keyserver", err)
+		}
+		conn, err := r.Dial(c)
+		if err != nil {
+			log.Fatal("Could not establish connection", err)
+		}
+		rpc := conn.RPC()
+		certfile, err := os.Open(delegateCert)
+		if err != nil {
+			log.Fatal("Could not open cert we want to get delegation from", err)
+		}
+		pemblock, err := ioutil.ReadAll(certfile)
+		if err != nil {
+			log.Fatal("Could not glob file", err)
+		}
+		certblock, _ := pem.Decode(pemblock)
+		cert, err := x509.ParseCertificate(certblock.Bytes)
+		if err != nil {
+			log.Fatal("Could not parse certificate", err)
+		}
+
+		ttl := 24 * time.Hour
+		cred, _, err := delegated.NewCredential(uint16(tls.ECDSAWithP256AndSHA256), delegated.VersionTLS13, ttl)
+		req, err := cred.Marshal()
+		if err != nil {
+			log.Fatal("failed to create query", err)
+		}
+		var resp []byte
+		err = rpc.Call("Delegator.Sign", req, &resp) //Richer queries and responses would be a good idea
+		if err != nil {
+			log.Fatal("failed in RPC", err)
+		}
+		dc, err := delegated.UnmarshalDelegatedCredential(resp)
+		if err != nil {
+			log.Fatal("failed to parse response")
+		}
+		valid, err := dc.Validate(cert, time.Now())
+		if err != nil {
+			log.Fatal("Error in validation", err)
+		}
+		if !valid {
+			log.Fatal("Got invalid response")
+		}
+		log.Fatalf("Got valid response")
+		log.Fatalf("Got response %v", resp)
```

Comment:
Only the first Fatalf is ever going to come through, and the script is going to exit with status 1. Should be 0 if successful.

fmt.Printf
fmt.Printf
return